azure-stack/hci/upgrade/install-enable-network-atc.md

```diff
@@ -51,21 +51,21 @@ In this step, you install Network ATC on every node in the cluster using the fol
 Install-WindowsFeature -Name NetworkATC
 ```
 
-### Step 2: Pause one node in the cluster
+### Step 2: Stop the Network ATC service
```

Comment:
@helmste - I will need to check with the PM on switching this step.

Reply:
@ronmiab - the reason for switching the steps is because the step 1 of installing the Network ATC feature and then stopping the service need to be performed on all nodes.  If we make this the second step, the third step of pausing one node of the cluster and starting to update the configuration is done on just one node.  It prevents the customer from jumping back and forth from all nodes to one node back to all nodes then back to one node.